.github/workflows/add-netlify-links.yml

```diff
@@ -1,55 +1,75 @@
 name: Add Netlify Links To Changed Pages
 on:
-    workflow_call:
+  workflow_call:
+  pull_request_target:
```

Comment:
Is this needed? I think it's just if you want to specify a particular branch, which we don't want in this case.

Reply:
Which part do you mean? I removed `pull_request_target` this morning after Nic let me know about the vulnerability, just to close the door on it in the meantime. This PR adds it back.

Reply:
I see, OK.